.asf.yaml

```diff
@@ -129,7 +129,7 @@ notifications:
   issues: commits@cloudberry.apache.org
   # Route pull request notifications to commits mailing list
   pullrequests: commits@cloudberry.apache.org
-  # Route discussion notifications to commits mailing list
-  discussions: commits@cloudberry.apache.org
+  # Route discussion notifications to dev mailing list
+  discussions: dev@cloudberry.apache.org
```

Comment:
It's better to route the GitHub discussions to @dev since not too many are posted now. Community members can also use the mailing list to reply to the discussion directly.